src/fiatconnect/slice.ts

```diff
@@ -47,11 +48,19 @@ export interface CachedQuoteParams {
   fiatType: FiatType
 }
 
+export interface CachedTransferDetails {
```

Comment:
Is it potentially worth it to include the `transactionHash` itself in the `CachedTransferDetails` interface? I know it's used as a key to the `cachedTransfers` map, but it may be useful so that callers don't have to keep track of `transactionHash` separately.

Reply:
Added it!